crates/bevy_winit/src/lib.rs

```diff
@@ -158,6 +158,21 @@ fn change_window(
                         y: position[1],
                     });
                 }
+                bevy_window::WindowCommand::Center => {
+                    let window = winit_windows.get_window(id).unwrap();
+
+                    // What to do if current_monitor is None?
+                    // Abort?
+                    // Or use primary_monitor? And then what if that also is None?
+                    let screen_size = window.current_monitor().unwrap().size();
```

Comment:
I think that the conservative thing to do here is to return early with a warning. There's no way to bubble that error back up (yet), but we can at least log it and not panic.